### PR TITLE
chore: Add empty rls policies CTA

### LINKED ISSUE #5812: RLS policy empty state is confusing
# Bug report

## Describe the bug

Empty state for RLS policies does not indicate that you need tables in public schema

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Create new project
2. Go to auth > policies

## Expected behavior

A clear and concise CTA to create a table in public schema, rather than the generic "Hmm, we couldn't find any results that matches your query. Try another?"

## Screenshots
Screenshot 2022-03-05 at 1 42 57 PM

## System information

- OS: [e.g. macOS, Windows]
- Browser (if applies) [e.g. chrome, safari]
- Version of supabase-js: [e.g. 6.0.2]
- Version of Node.js: [e.g. 10.10.0]

### PR DESCRIPTION
**Pull request by louis-27**
## What kind of change does this PR introduce?

Adds CTA for users to create a table in the public schema.

Closes #5812 

## What is the current behavior?

SCR-20220308-adn

## What is the new behavior?

https://user-images.githubusercontent.com/48682663/157141965-5df333d2-0d67-4058-9088-3d6618b89027.mov

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: studio/components/to-be-cleaned/NoTableState.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { FC } from 'react'
+import { useRouter } from 'next/router'
+import ProductEmptyState from './ProductEmptyState'
+
+interface Props {
+  message: string
+}
+
+const NoTableState: FC<Props> = ({ message }) => {
```
Comment (Ziinc) [inline studio/components/to-be-cleaned/NoTableState.tsx:9]: React.FC

File: studio/components/to-be-cleaned/NoTableState.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { FC } from 'react'
```
Comment (Ziinc) [inline studio/components/to-be-cleaned/NoTableState.tsx:1]: no need to import

**Review by Ziinc — CHANGES_REQUESTED** [review]
Thanks for the contrib. Instead of placing the abstracted component under the `to-be-cleaned` folder, could you put it under `components/ui/States/NoTableState.tsx`

small nitpicks, looks good.

File: studio/components/interfaces/Database/Triggers/CreateTrigger.tsx [hunk 3]
```diff
@@ -373,33 +373,13 @@ const CreateTrigger: FC<CreateTriggerProps> = ({ trigger, visible, setVisible })
             </CreateTriggerContext.Provider>
           </div>
         ) : (
-          <NoTableState />
+          <NoTableState message="You will need to create a table first before you can make a trigger" />
```
Comment (joshenlim) [inline studio/components/interfaces/Database/Triggers/CreateTrigger.tsx:376]: any reason why we shouldn't use ProductEmptyState directly, and instead creating a NoTableState as a wrapper around that? just feels unnecessary

Comment (louis-27) [reply studio/components/interfaces/Database/Triggers/CreateTrigger.tsx:376]: hi, sorry for the late reply. Previously, the CreateTrigger component was doing NoTableState, and from this issue, the  auth/policies.tsx page needed to do the same (but with a different message). So I decided to make a separate component for NoTableState to be used by CreateTrigger and the auth/policies.tsx page

File: studio/components/to-be-cleaned/NoTableState.tsx (hunk 2 above)
Comment (louis-27) [reply studio/components/to-be-cleaned/NoTableState.tsx:1]: oh okay, thanks for the suggestion! just made the changes 👍

**Review by Ziinc — APPROVED** [review]
🙏

**Pull request merged**